1.0.2/build-msbuild/Dockerfile

```diff
@@ -0,0 +1,56 @@
+# TODO pull from microsoft/dotnet when these become available
+FROM microsoft/dotnet-nightly:1.0.2-sdk-msbuild
```

Comment:
I don't think we should be producing a 1.0.2-sdk-msbuild image.  This workaround should be contained as much as possible IMO.  I would go so far as ask if you can just retag 1.0.1 as 1.0.2 and not call it out in the readme?